bootloader/src/old_machine_bootloader_linux.c

```diff
@@ -26,17 +26,18 @@ struct response {
 void init_response(struct response *s) {
   s->len = 0;
   s->ptr = malloc(s->len+1);
-  if (s->ptr == NULL)
+  if (NULL == s->ptr) {
```

Comment:
What? Does this matter?

Reply:
https://en.wikipedia.org/wiki/Yoda_conditions

Reply:
Cha, nice! Didn't knew that!